Program.cs

```diff
@@ -35,9 +38,18 @@ private Task Log(LogMessage msg)
 
         private async Task MessageReceived(SocketMessage msg)
         {
-            if (msg.Content == "!gimmewater")
+            switch (msg.Content)
             {
-                await msg.Channel.SendMessageAsync("/tts here's ur water u thirsty hoe 💦");
+                case "!gimmewater":
+                    await msg.Channel.SendMessageAsync("/tts here's ur water u thirsty hoe 💦");
+
+                    Console.WriteLine("Starting timer");
+                    BotTimer.Start((SocketTextChannel)msg.Channel);
+                    break;
+                case "!stopplsnomore":
```

Comment:
I would pick all these from a Constants.cs file maybe. Cleaner code and single place for all vars